docs/configuration.rst

```diff
@@ -19,6 +19,8 @@ General Settings
     Twilio_.
   * ``'two_factor.gateways.fake.Fake'``  for development, recording tokens to the
     default logger.
+  * ``'two_factor.gateways.fake.QueryableFake'``  for testing, recording tokens
+    to the ``QueryableFake`` singleton.
```

Comment:
The new gateway is not a singleton. I know what you're trying to say here (`sms_tokens` and `call_tokens` are initialised when the class is created not when an object is instantiated) but "singleton" would mean that  things like `QueryableFake().sms_tokens = {}` would affect other instances of `QueryableFake` because there would only be one instance of the class.